tests/Relay.stub

```diff
@@ -0,0 +1,1796 @@
+<?php
+
+/**
+ * @generate-function-entries
+ * @generate-legacy-arginfo
+ */
+
+namespace Relay;
+
+class Relay {
+    public const OPT_COMPRESSION = 1;
+    public const OPT_COMPRESSION_LEVEL = 1;
+    public const OPT_PHPREDIS_COMPATIBILITY = 1;
+    public const OPT_PREFIX = 1;
+    public const OPT_READ_TIMEOUT = 1;
+    public const OPT_SCAN = 1;
+    public const OPT_SERIALIZER = 1;
+    public const SERIALIZER_PHP = 1;
```

Comment:
It would be nice to fix these constants.

Reply:
Fix how?

Reply:
I've made up all of them based on PHPStan reporting.
Their value are not `1` for sure!